packages/ui-color-picker/src/ColorIndicator/README.md

```diff
@@ -0,0 +1,27 @@
+---
+describes: ColorIndicator
+---
+
+A component displaying a circle with checkerboard background capable of displaying colors
+
+### Color Indicator
+
+```js
+---
+example: true
+---
+<ColorIndicator color='#ff0000'/>
+
+
+```
+
+### Color Button Pattern
+
+```js
+---
+example: true
+---
+<IconButton>
+  <ColorIndicator color='#ff0000'/>
+</IconButton>
```

Comment:
This example generates an error in the console about `ScreenReaderLabel` missing

Reply:
Added